src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/actions/WorkOnBacklog.kt

```diff
@@ -31,7 +32,7 @@ class WorkOnBacklog private constructor(
     }
 
     private val logger: Logger = LogManager.getLogger(this::class.java)
-    private val newSprintName = "New Sprint Name"
+    private val newSprintName = RandomStringUtils.randomAlphabetic(10)
```

Comment:
no need to include apache-commons-lang library just to generate random string.
It doesn't need to be random, it needs to be unique:
`"New Sprint Name - " + System.currentTimeMillis()`

Reply:
I picked RandomStringUtils because it is already used in similar cases in iJPT, but I'm okay with using your approach. It is similar to the one I tried at the beginning (UUID) but it solves the problem of a too long name.